#!/bin/sh /usr/share/dpatch/dpatch-run
## 50_hp-clean_fix.dpatch by Henrique de Moraes Holschuh <hmh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix wipeAndSpitType1() invocation

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./clean.py ./clean.py
--- ./clean.py	2005-09-17 00:43:27.000000000 -0300
+++ ./clean.py	2005-09-17 13:22:55.147332326 -0300
@@ -143,7 +143,7 @@
 
 if clean_type in (CLEAN_TYPE_PCL,CLEAN_TYPE_PCL_WITH_PRINTOUT):
     if level == 3:
-        maint.wipeAndSpitType1()
+        maint.wipeAndSpitType1( d )
     elif level == 2:
         maint.primeType1( d )
     else:
